Spigot-Server-Patches/0546-Add-Destroy-Speed-API.patch

```diff
@@ -0,0 +1,40 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Ineusia <ineusia@yahoo.com>
+Date: Sat, 18 Jul 2020 18:49:56 -0500
+Subject: [PATCH] Add Destroy Speed API
+
+
+diff --git a/src/main/java/org/bukkit/craftbukkit/block/CraftBlock.java b/src/main/java/org/bukkit/craftbukkit/block/CraftBlock.java
+index 9b0e868f006a34b47ef9a48c7ea844be8122c291..dc5a253cd03ccdddedba8cd48005bb62568aa4ca 100644
+--- a/src/main/java/org/bukkit/craftbukkit/block/CraftBlock.java
++++ b/src/main/java/org/bukkit/craftbukkit/block/CraftBlock.java
+@@ -5,6 +5,7 @@ import java.util.Collection;
+ import java.util.Collections;
+ import java.util.List;
+ import java.util.stream.Collectors;
++
+ import net.minecraft.server.AxisAlignedBB;
+ import net.minecraft.server.BiomeBase;
+ import net.minecraft.server.BlockPosition;
+@@ -16,6 +17,7 @@ import net.minecraft.server.EnumSkyBlock;
+ import net.minecraft.server.GeneratorAccess;
+ import net.minecraft.server.IBlockData;
+ import net.minecraft.server.IRegistry;
++import net.minecraft.server.Item;
```

Comment:
```suggestion
```

Reply:
doesnt look like this belongs here, as dont even see it used.

Reply:
leftover from putting JRoy's suggestion into the patch properly, evidently im very bad at watching imports